Sky Atlas/Sky Atlas.html

```diff
@@ -43726,19 +44252,155 @@ <h1>Agent Scope Database</h1>
+          </td>
+          <td>Grove</td>
+          <td>Core</td>
+          <td>The signers of the Prime Relayer Multisig must use the Multisig to exercise the <code>RELAYER_ROLE</code> in accordance with the instructions specified in the Grove Artifact.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>Prime Relayer Multisig - Modification</dfn>
+          </td>
+          <td>Grove</td>
+          <td>Core</td>
+          <td>Grove can change the signers of the Prime Relayer Multisig at any time, so long as there are at least two (2) signers and at least a majority of signers are required to execute transactions.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>Core Operator Relayer Multisig</dfn>
           </td>
           <td>Grove</td>
           <td>Core</td>
-          <td>The Core Operator Relayer Multisig has the <code>RELAYER_ROLE</code> as defined in <dfn>Relayer Role</dfn>. In a future iteration of the Grove Artifact, additional logic will be added herein.</td>
+          <td>The Core Operator Relayer Multisig has the <code>RELAYER_ROLE</code> as defined in <dfn>Relayer Role</dfn> and is controlled by Amatsu.</td>
         </tr>
+        <tr>
+          <td>
+            <dfn>Core Operator Relayer Multisig - Address</dfn>
+          </td>
+          <td>Grove</td>
+          <td>Core</td>
+          <td>The address of the Core Operator Relayer Multisig on the Ethereum Mainnet is <code>0x4364D17B578b0eD1c42Be9075D774D1d6AeAFe96</code>.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>Core Operator Relayer Multisig- Required Number Of Signers</dfn>
+          </td>
+          <td>Grove</td>
+          <td>Core</td>
+          <td>The Core Operator Relayer Multisig currently has a 2/3 signing requirement.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>Core Operator Relayer Multisig - Signers</dfn>
+          </td>
+          <td>Grove</td>
+          <td>Core</td>
+          <td>The signers of the Core Operator Relayer Multisig are three (3) addresses controlled by Operational GovOps Amatsu.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>Core Operator Relayer Multisig - Usage Standards</dfn>
+          </td>
+          <td>Grove</td>
+          <td>Core</td>
+          <td>The signers of the Core Operator Relayer Multisig must use the Multisig to exercise the <code>RELAYER_ROLE</code> in accordance with the instructions specified in the Grove Artifact.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>Core Operator Relayer Multisig - Modification</dfn>
+          </td>
+          <td>Grove</td>
+          <td>Core</td>
+          <td>Operational GovOps Amatsu can change the signers of the Core Operator Relayer Multisig at any time, so long as there are at least three (3) signers and at least two thirds of signers are required to execute transactions.</td>
+        </tr>
+        <tr>
         <tr>
           <td>
```

Comment:
duplicate opening tag

```suggestion
          <td>Operational GovOps Amatsu can change the signers of the Core Operator Relayer Multisig at any time, so long as there are at least three (3) signers and at least two thirds of signers are required to execute transactions.</td>
        </tr>
        <tr>
          <td>
```